scripts/create_elmo_embeddings_from_vocab.py

```diff
@@ -0,0 +1,87 @@
+# pylint: disable=no-self-use
+
+import os
+import argparse
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir))))
+
+import h5py
+import numpy
+import torch
+from torch.autograd import Variable
+
+from allennlp.data.token_indexers import ELMoTokenCharactersIndexer
+from allennlp.modules.elmo import _ElmoCharacterEncoder
+from allennlp.data import Token, Vocabulary
+
+
+def main(vocab_path: str,
+         elmo_model_dir: str,
+         output_dir: str,
+         batch_size: int,
+         device: int):
+
+    options_file = os.path.join(elmo_model_dir, 'options.json')
```

Comment:
These file names will change between models to be more descriptive, so we need to pass in the individual names not the directory.

Reply:
sure